src/main/scala/com/gu/invoicing/preview/Model.scala

```diff
@@ -48,16 +54,18 @@ object Model extends JsonSupport {
     productName: String,        /* For example Newspaper Delivery                          */
     chargeName: String,         /* For example Sunday                                      */
     dayOfWeek: DayOfWeek,       /* Strongly typed day of publication                       */
-    price: Double,              /* Charge of single publication                            */
+    price: Double,              /* Charge of single publication (including tax)            */
```

Comment:
I'm always a bit worried about representing money as a double - BigDecimal always seems to be safer - https://stackoverflow.com/questions/3413448/double-vs-bigdecimal